challenge.js

```diff
@@ -54,8 +53,8 @@
  *		use to achieve this.
  *
  *	Paste the URL of the MDN page below.
- */
+ */https://developer.mozilla.org/en-US/docs/Web/API/Node/appendChild
```

Comment:
That's right. Here's an example of it in action:

Correct. All HTML elements are Nodes; if `div` is a reference to an on-DOM div, and we have a off-DOM element `e` we created in our JS, we can attach it with this method:

```js
var div = document.querySelectorAll("body > div")[0]; // the zeroth element of the set of all divs that are immediate children of the body
var e = document.createElement('textarea'); // this textarea element is not on the DOM yet
div.appendChild(e); // mission accomplished
```

1